strings/strings/jp.yml

```diff
@@ -0,0 +1,529 @@
+name: Japanese
+natively: Japan
+
+authors:
+  - Achu biju (AmiFutami)
+
+# commons
+com_1: "`処理...`"
+com_2: "`検索...`"
+com_3: "`無効入力`"
+com_4: "`サポートされていないメディア..`"
+com_5: "`ダウンロード...`"
+com_6: "`アップロード...`"
+
+# chatactions
+can_1: "#GBanned_User 参加しました.\n\n**User** : {}\n**Reason**: {}\n\n`ユーザー禁止.`"
+can_2: "∆ #UsernameUpdate\n\n@{} changed username to @{}"
+can_3: "∆ #UsernameUpdate\n\n{} removed its username. (@{})"
+can_4: "∆ #UsernameUpdate\n\n{}'s new username --> @{}"
+
+# help
+help_1: "`{}` is not a valid plugin!"
+help_2: "ボットはインラインクエリに応答しませんでした.\n使用を検討してください `{}restart`"
+help_3: "`からボットのインラインモードをオンにしてください` @Botfather."
+help_4: "• プラグイン"
+help_5: "• アドオン"
+help_6: "••ボイスチャット"
+help_7: "インラインプラグイン••"
+help_8: "⚙️ オーナ ツール
+help_9: "設定 ⚙️"
+help_10: "••Cʟᴏꜱᴇ••"
+help_11: "Plugin Name-{}\n\n✘ Commands Available -\n\n"
+
+
+# userlogs
```

Comment:
strings (" ") are missing here, making it invalid yaml file.
Can you please fix it